documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -18,6 +18,11 @@ This document defines guidelines for telco operator exposure platforms to manage
     - [Authorization flows / grant types](#authorization-flows--grant-types)
       - [Authorization code flow (Frontend flow)](#authorization-code-flow-frontend-flow)
```

Comment:
```suggestion
Detailed information on the below specified grant types (for eg. extensions to some of the grant types, links to original RFC documents etc.) is available in [CAMARA-AuthN-AuthZ-Concept.md](https://github.com/camaraproject/IdentityAndConsentManagement/blob/main/documentation/CAMARA-AuthN-AuthZ-Concept.md)
      - [Authorization code flow (Frontend flow)](#authorization-code-flow-frontend-flow)
```

Reply:
Fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/93/commits/d20fe33f6f8c02db75f6475acb18cd65aae8c986